src/snyk/common/configuration/configuration.ts

```diff
@@ -171,6 +171,11 @@ export class Configuration implements IConfiguration {
     return `${authUrl.toString()}manage/snyk-code?from=vscode`;
   }
 
+  get snykLanguageServerPath(): string {
```

Comment:
should this be hidden behind a feature flag param? If so, what should I use?

Reply:
This isn't referenced anywhere at the moment, thus no hiding should happen. Once @bastiandoetsch wires language server this config will be accessed only behind the feature flag, so all good here ✅